ProcessLib/ThermoMechanicalPhaseField/ThermoMechanicalPhaseFieldFEM-impl.h

```diff
@@ -439,7 +439,7 @@ void ThermoMechanicalPhaseFieldLocalAssembler<ShapeFunction, IntegrationMethod,
         double const T_ip = N.dot(T);
         double const delta_T = T_ip - T0;
         // calculate real density
-        double const rho_s = rho_sr * (1 - 3 * alpha * delta_T);
+        double const rho_s = rho_sr / (1 + 3 * alpha * delta_T);
```

Comment:
Why is this change not reflected in the ctests? There must be a significant difference, isn't it?

Reply:
1. body force is not considered in current testing cases.
2. under the present values of parameters, the density does not have large changes.